specification/app/resource-manager/Microsoft.App/preview/2024-08-02-preview/Jobs.json

```diff
@@ -921,7 +921,7 @@
             "runningState": {
               "description": "Current running state of the job",
               "enum": [
-                "Running",
+                "Ready",
```

Comment:
is this also updated in the Control Plane PR?

Reply:
not yet i'm making the change currently and will push it later today. integration tests will have to wait for this PR to merge and for the internal sdk to generate with the new "Ready" change